### ISSUE TITLE
Using LUIS for entity extraction: found entity won't write to property

### ISSUE THREAD

**Issue by dfvr1994** [microsoft/BotFramework-Composer#2554]
## Describe the bug

When running the updated tutorial here [https://docs.microsoft.com/en-us/composer/tutorial/tutorial-luis#using-luis-for-entity-extraction](url), after completing all the 5 steps, the bot doesn't seem to write the recognized entity "@zipcode" to the property "user.zipcode". 

## Screenshots

I notice that the "Set a property" action looks different to the one in the tutorial, notice the " : = @ " instead of " = =@ ". I don't know how to set it to the one on the tutorial. 

![image](https://user-images.githubusercontent.com/45764991/78742617-2aac3800-7922-11ea-8fea-ad5b5e2622be.png)

This is the response from Luis:

![image](https://user-images.githubusercontent.com/45764991/78743141-63004600-7923-11ea-81a4-e47412cb9bc3.png)

![image](https://user-images.githubusercontent.com/45764991/78743371-06e9f180-7924-11ea-9c98-ffee36f16254.png)

**Comment by hibrenda** [comment]
@dfvr1994 please try to use "=@zipcode" as following. we will update the document .
![image](https://user-images.githubusercontent.com/43408889/78750947-dcd81400-79a3-11ea-9f1c-f22daece0703.png)

**Comment by dfvr1994** [comment]
@hibrenda I did have the Value set to "=@zipcode" 

![image](https://user-images.githubusercontent.com/45764991/78801038-c3be6b80-7981-11ea-9a96-63d6290c902d.png)

I noticed that my interface doesn't show the "boolean" and "expression" dropdown menus, also the "fx" indicators, am I using a different or older version? Is that why the found entity won't write to property?

**Comment by zxyanliu** [comment]
Hi @dfvr1994 Thanks for reporting. I just tested and verified the instructions in the [Using LUIS Tutorial](https://docs.microsoft.com/en-us/composer/tutorial/tutorial-luis). The steps work on my side. 

I think I know where the problem is, could you please double check the example utterances you used to define your intent using LUIS recognizer? This is what I used. 

![image](https://user-images.githubusercontent.com/32497439/78816758-05054a00-7987-11ea-9c4c-157f7030527c.png)

In the instruction of the [use luis for entity extraction](https://docs.microsoft.com/en-us/composer/tutorial/tutorial-luis#using-luis-for-entity-extraction) section, you need to do the following: 
- Add the entity definition in regex to your weather intent defined in LUIS. See instruction in [this section](https://docs.microsoft.com/en-us/composer/tutorial/tutorial-luis#using-luis-for-entity-extraction) and in step 1. 
- Add a `Set a property` action and set the `user.zipcode` property value to `=@zipcode`. This means when you enter a valid zipcode together with your utterance, the bot will no longer prompt for it and the flow will continue with the `HTTP` request.  
- When testing this, we recommend you use the utterance `how is the weather in` + `a valid zipcode`. For example, try enter `how is the weather in 98004` in the Emulator. 

Finally, it's a nice catch that the screenshot of the `Set a property` view in the authoring canvas is not updated. The `:=` instead of `==` is a known bug and I was unaware it's already fixed. Anyway, I have updated this screenshot in the doc, which will go live soon. 

Could you please try again and let us know your updates? 

Thanks!

**Comment by dfvr1994** [comment]
Hi @zxyanliu , thank you for answering :D 

I double checked the entity definition in regex, here it is: 

![image](https://user-images.githubusercontent.com/45764991/78837580-ff752780-79b9-11ea-9e16-98e2a814829e.png)

Do I have to place it under the weather intent?

Also, I added a response to the bot, just to see if it is catching the zipcode and this is what I got: 

1/2
![image](https://user-images.githubusercontent.com/45764991/78837771-5d097400-79ba-11ea-8d09-a318ea5c889d.png)

2/2
![image](https://user-images.githubusercontent.com/45764991/78837810-7b6f6f80-79ba-11ea-9013-9241ac9367e5.png)

Response: 
![image](https://user-images.githubusercontent.com/45764991/78838014-ddc87000-79ba-11ea-8331-f50fe46412a3.png)

I think Luis is catching the entity right?

**Comment by zxyanliu** [comment]
Hi @dfvr1994 
I think the intent definition is fine, but your testing in the Emulator did not show the expected response from the `HTTP` request. Let me double check two things: 
1. Did you complete the steps in this [doc](https://docs.microsoft.com/en-us/composer/tutorial/tutorial-get-weather), especially the [seciton](https://docs.microsoft.com/en-us/composer/tutorial/tutorial-get-weather#add-an-http-request) to add an `HTTP` request? 

2. Let me share the flow in the authoring canvas. I wish I can take a whole screenshot but I cannot, so I split them. Please check. 

    - First half of the `BeginDialog` flow. 

     ![image](https://user-images.githubusercontent.com/32497439/78839138-a05ce600-79ac-11ea-8c08-8f36ec607a75.png)

     - Second half of the `BeginDialog` flow. 

     ![image](https://user-images.githubusercontent.com/32497439/78838846-e8c7d400-79ab-11ea-8ce8-56a21d378cd2.png)

**Comment by dfvr1994** [comment]
Hi @zxyanliu I think I found the error, 

Based on the text at the end of step 3 of the tutorial, I set the `Always prompt` property of the Text Input action to `true`.

![image](https://user-images.githubusercontent.com/45764991/78844216-c3968e00-79ca-11ea-8528-25ea14971a88.png)

![image](https://user-images.githubusercontent.com/45764991/78844402-428bc680-79cb-11ea-9898-ea6ad921a122.png)

This caused the bot to ask for the `zipcode` even though Luis got the entity right.

I set the `Always prompt` property of the Text Input action to `false` and got the bot working the way it is supposed to.

![image](https://user-images.githubusercontent.com/45764991/78844886-b37fae00-79cc-11ea-8c37-0dbd83a5e17c.png)

Maybe It could be a good idea to add some kind of hint on step 7 of the tutorial, so that the user toggles the `Always prompt` property of the Text Input action to `false`.
